### PR TITLE
Atlantic 2024 talks

### PR DESCRIPTION
**Pull request by plaindocs**
- [x] missing a few pics
- [x] missing blog post
- [x] missing updated CTA links

----
📚 Documentation preview 📚: https://writethedocs-www--2156.org.readthedocs.build/

### CODE REVIEW

File: docs/conf/atlantic/2024/news/announcing-speakers.rst [hunk 1]
```diff
@@ -0,0 +1,56 @@
+:template: {{year}}/generic.html
+
+.. post:: June 06, 2024
```
Comment (CollierCZ) [inline docs/conf/atlantic/2024/news/announcing-speakers.rst:3]: Is this intentionally tomorrow? Just to check if it appears correctly.

File: docs/conf/atlantic/2024/news/announcing-speakers.rst [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+Head over to the `Speakers page <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/speakers/>`_ to see the full abstracts.
+We hope there’s something there for everyone to enjoy, and also something that will broaden your horizons a bit.
+
+Tickets are going fast!
+-----------------------
+
+If you don't have your ticket yet, now's a great time to snap one up. Conference tickets have sold out for the last several years, and they go faster along with the speaker announcement, so if you're planning to come, don't leave it to the last minute.
+
+We will close registrations around one week before the conference, to give folks enough time to set up their Hopin credentials.
```
Comment (CollierCZ) [inline docs/conf/atlantic/2024/news/announcing-speakers.rst]: I believe we should not be mentioning Hopin.
```suggestion
We will close registrations around one week before the conference, to give folks enough time to set up their credentials for the conference platform.
```

File: docs/conf/atlantic/2024/news/announcing-speakers.rst [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+You can apply until **{{ grants.ends }}** on `our website <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/opportunity-grants/>`_.
+
+Thanks to our sponsors
+----------------------
+
+We are so grateful to have our sponsors help in bringing our events to life every year.
+
+Head over to our `sponsorship prospectus <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/sponsors/prospectus/>`_ for details on how your company can get involved!
+
+We have several types of sponsorship available, including tables at the job fair that we'll be hosting on Tuesday.
```
Comment (CollierCZ) [inline docs/conf/atlantic/2024/news/announcing-speakers.rst]: "Tables" feels too physical to me, but maybe that's just a personal thing.
```suggestion
We have several types of sponsorship available, including spaces at the job fair that we'll be hosting on Tuesday.
```

File: docs/_data/atlantic-2024-sessions.yaml [hunk 4]
```diff
@@ -0,0 +1,427 @@
...
+    <li>Prioritize content delivery so the team does not face burnout and still delivers
+    the most helpful content to our users.</li>
+
+    <li>Deliver content that covers the user''s knowledge needs.</li>
+
+    </ul>
+
+    <p>The cherry on top is that we will share a usable template of our documentation
+    plan that you can readily use for your documentation projects.</p>'
+- title: 'From morphemes to manuscripts: how linguistics can make you a better writer'
```
Comment (CollierCZ) [inline docs/_data/atlantic-2024-sessions.yaml]: I understand we don't force people into either title or sentence case (they can choose), but can we at least capitalize subtitles?
```suggestion
- title: 'From morphemes to manuscripts: How linguistics can make you a better writer'
```

File: docs/conf/atlantic/2024/news/announcing-speakers.rst [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+
+Greetings, documentarians! Today we're excited to announce that we've finalized our speakers for the next virtual {{city}} conference!
+
+Thank you so much to everyone who took the time to send us a proposal – we literally wouldn't have an event without you!
+
+Full speaker line-up
+--------------------
+
+Every year, we look to bring a wide range of voices to the Write the Docs stage. 
+Because the role of "documentarian" looks so different to each of us, we aim for a line-up that spans a good mix of practical, philosophical, and technical topics – with the odd whimsical one thrown in for good measure.
```
Comment (CollierCZ) [inline docs/conf/atlantic/2024/news/announcing-speakers.rst]: Is there a topic that the description "odd whimsical" especially applies to this time? I did not think so at quick glance.

**Review by CollierCZ — COMMENTED** [review]
I'm no expert on conference pages, but looks good overall. Some style questions thrown in to the review.

The pictures that are taller than they are wide get pressed into shape for the speaker page and look rather awkward. Would it be possible to crop them to more of a square shape or change the image processing?

**Comment by plaindocs** [comment]
Thanks @CollierCZ - we could improve the pics for sure, but they are uploaded by the speakers themselves, and so can be updated.

@mxsasha any ideas on improving how we convert the pics? Or do we just grab as is from Pretalx.

**Comment by plaindocs** [comment]
This is good to go if someone wants to :+1:

**Review by CollierCZ — APPROVED** [review]

**Comment by mxsasha** [comment]
> @mxsasha any ideas on improving how we convert the pics? Or do we just grab as is from Pretalx.

Hmm, never noticed this issue before. We just take the image as is. If we/they don't like how it looks, the best result is probably by them uploading a better photo.

**Comment by plaindocs** [comment]
> > @mxsasha any ideas on improving how we convert the pics? Or do we just grab as is from Pretalx.
> 
> Hmm, never noticed this issue before. We just take the image as is. If we/they don't like how it looks, the best result is probably by them uploading a better photo.

Seems fine. File as no action needed. :-D

**Pull request merged**